BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -37,21 +38,36 @@ public struct ElementContent {
     ///   - environment: The environment to measure in.
     /// - returns: The layout size needed by this content.
     public func measure(in constraint: SizeConstraint, environment: Environment) -> CGSize {
+        return measure(
+            in: constraint,
+            environment: environment,
+            cache: CacheFactory.makeCache(name: "ElementContent")
+        )
+    }
+
+    func measure(in constraint: SizeConstraint, environment: Environment, cache: CacheTree) -> CGSize {
```

Comment:
Should we combine these two measure functions into one with a default value for the cache? E.g. 
```swift
cache: CacheTree = CacheFactory.makeCache(name: "ElementContent")
```